### PR TITLE
teamcity-trigger: pass TAGS for deadlock builds

### PR DESCRIPTION
**Pull request by tamird**

### CODE REVIEW

File: cmd/post-github/main.go [hunk 1]
```diff
@@ -59,7 +54,7 @@ func runGH(
 
 Stress build found a failed test:
 
-%s`, sha, makeCodeLike(test.Message))
```
Comment (tbg) [inline cmd/post-github/main.go:62]: probably paranoia, but technically escaping ``` is needed.

**Review by tbg — APPROVED** [review]
LGTM

File: cmd/post-github/main.go (hunk 1 above)
Comment (tamird) [reply cmd/post-github/main.go:62]: Agreed, and yet there appears to be no sane way to do it. It's also the case that "```" is only recognized as closing a code block if it's on a line by itself...and then it's not clear how to escape it without changing it. Gonna punt.

Comment (bdarnell) [reply cmd/post-github/main.go:62]: You can't escape the triple-backticks, but there's another way to make a code block: indent every line by four spaces.

Comment (tamird) [reply cmd/post-github/main.go:62]: Ack. Will revisit if it's a problem.

**Pull request merged**